interop/javapoet/src/main/kotlin/com/squareup/kotlinpoet/javapoet/typeAliases.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.kotlinpoet.javapoet
+
+/*
+ * Useful typealiases for colliding names
+ */
```

Comment:
nit (I believe as is this would be attached as doc for `KTypeName` which seems wrong):
```suggestion
// Useful typealiases for colliding names
```

Reply:
It actually won't be attached to any elements if it's just a single star statement like this. It only becomes a full kdoc if you have two stars (`/**`)

Reply:
Ah, true, nvm then!